data/json/items/containers.json

```diff
@@ -1133,6 +1148,27 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ], [ "BOIL", 2 ] ]
   },
+  {
+    "id": "jug_plastic_sealed",
+    "copy-from": "jug_plastic"
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "sealed gallon jug" },
+    "looks_like": "bottle_plastic",
+    "description": "A standard plastic jug used for milk and household cleaning chemicals. This one still has an intact tamper-evident seal.",
```

Comment:
Same thing, it can't mention the state of the seal.

Reply:
This one is done too.